src/test/java/com/exasol/matcher/AbstractCellValueMatcherTest.java

```diff
@@ -167,4 +166,19 @@ void testFuzzyMatchTwoStrings() {
     void testFuzzyMismatchTwoStrings() {
         assertTypeFuzzyMismatch("VARCHAR(40)", "'a'", "b");
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = { "26.81", "26.83" })
+    void testFuzzyMatchDecimalWithTolerance(final String actualValue) {
+        assertTypeFuzzyMatch("DECIMAL(30,3)", actualValue,
+                FuzzyCellMatcher.fuzzilyEqualTo(26.82, BigDecimal.valueOf(0.015)));
```

Comment:
```suggestion
                FuzzyCellMatcher.fuzzilyEqualTo(26.82, BigDecimal.valueOf(0.015)));
```

When looking at this, I think we should have convenience methods that accept non-big-int tolerances.

Reply:
I'm not sure if it's a good idea to define tolerance as floating-point numbers. I think tolerance should be very exact.

Reply:
On the other hand's side this constructor of BigDecimal also uses a double as input...

Reply:
I accept the precision argument. Continue.